docs/proposals/keeping-transient-pod-name-spaces-alive.md

```diff
@@ -0,0 +1,54 @@
+# Keeping transient pod name spaces alive
+
+## Introduction
+The goal of this proposal is to give administrators options to keep namespaces and therefore APB pods after the execution. The use cases for this feature are demos and debugging APBs.
+
+## Problem Description
+The [bug](https://bugzilla.redhat.com/show_bug.cgi?id=1497766) was created when we moved to creating transient namespaces during execution of the APB pod. What happens  This bug creates issues for debugging APB's as well as issues with demos. 
+
+## Implementation Details.
+
+### Configuration Values
+
+* `keep_namespace` or `ClusterConfig.KeepNamespace` - Parameter to always keep name space no matter what.
+* `keep_namespace_on_error` or `ClusterConfig.KeepNamespaceOnError` -  parameter to keep name space around if an error occurs in the play book that is running.
+
+Example:
+```yaml
+...
+openshift:
+  ....
+  keep_namespace: false
+  keep_namespace_on_error: true
+...
+```
+
+**NOTE: Both will default to false in openshift-ansible, but will be set to true for keep_namespace_on_error**
```

Comment:
I don't quite understand the above NOTE. both will be false, but set to true for namespace_on_error. HUH? 😕

Reply:
Was totally in my one head on this one, will make it more clear